common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -184,8 +188,12 @@ def create_student(self, student_uid):
         """
         url = self.students_url
         payload = {'uid': student_uid}
-        response = requests.post(url, headers=self.request_headers, data=payload)
-        student = json.loads(response.content)
+        try:
+            student = {}
```

Comment:
@macornwell This should go into the [`except` block](https://github.com/open-craft/edx-platform/pull/69#discussion_r148497254).